Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -1514,15 +1515,21 @@ Messages::WebDriverClient::ElementClickResponse WebDriverConnection::element_cli
 }
 
 // 12.5.2 Element Clear, https://w3c.github.io/webdriver/#dfn-element-clear
-Messages::WebDriverClient::ElementClearResponse WebDriverConnection::element_clear(String const&)
+Messages::WebDriverClient::ElementClearResponse WebDriverConnection::element_clear(String const& element_id)
 {
     dbgln("FIXME: WebDriverConnection::element_clear()");
 
-    // FIXME: 1. If element's innerHTML IDL attribute is an empty string do nothing and return.
+    // 1. If element's innerHTML IDL attribute is an empty string do nothing and return.
+    Web::DOM::Element* element = TRY(get_known_connected_element(element_id));
+    if (!element) {
+        return Web::WebDriver::Error::from_code(Web::WebDriver::ErrorCode::NoSuchElement, "Element not found");
+    }
```

Comment:
Either implement the whole line or keep FIXME I'd say.

The code below is untested, but something like it is what the spec excepts.

```suggestion
    Web::DOM::Element* element = TRY(get_known_connected_element(element_id));
    if (!element) {
        return Web::WebDriver::Error::from_code(Web::WebDriver::ErrorCode::NoSuchElement, "Element not found");
    }
    if (!element->inner_html()) {
        return ""sv
    }
          
```